ecs-microservice/variables.tf

```diff
@@ -1,3 +1,8 @@
+variable "environment" {
```

Comment:
this is a new mandatory field, when upgrading to this and newer version of ecs-microservice module you will have to add this variable to your code defining a ecs_microservice resource.